prow/config/config.go

```diff
@@ -218,6 +226,9 @@ func parseConfig(c *Config) error {
 		return fmt.Errorf("parsing template: %v", err)
 	}
 	c.Plank.ReportTemplate = reportTmpl
+	if c.Plank.MaxConcurrency < 0 {
+		return fmt.Errorf("plank has invalid max_concurrency (%d), it needs to be a non-negative number", c.Plank.MaxConcurrency)
```

Comment:
Why not `uint`?

Reply:
Staying consistent with the ProwJob API and an API convention in Kube: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#primitive-types

> Do not use unsigned integers, due to inconsistent support across languages and libraries. Just validate that the integer is non-negative if that's the case.

Reply:
ack